lib/parser/word/entry/dictionary_entry/headword.rb

```diff
@@ -1,16 +1,29 @@
+require_relative '../../../../converter'
```

Comment:
The structure of the code is based on the XML produced by Morpheus. I'm still [not sure](https://github.com/perseids-tools/morpheus-perseids-api/commit/8dd26f152a986d03ae246d788c3441dbc59e8e38) if this is the best way to model the application. It results in these highly nested files. But the structure has made it easy to build the XML because each object only has to know about itself then call `.bamboo_xml` on its children.